lib/MediaInfo.js

```diff
@@ -483,7 +483,7 @@ MediaInfo.create = function(type,supported)
 			 }
 		 }
 		//Add extensions
-		for (let id = 0; supported.extensions && id<supported.extensions.length; ++id)
+		for (let id = 1; supported.extensions && id<supported.extensions.length; ++id)
 			//Add to answer
 			mediaInfo.addExtension(id, supported.extensions[id]);
```

Comment:
woudn't we be missing the first supported extension? I think the correct fix should be: 

```
		for (let i = 0; supported.extensions && id<supported.extensions.length; ++i)
			//Add to answer
			mediaInfo.addExtension(i+1, supported.extensions[i]);
```

What do you think?